packages/core/factories/src/useCartFactory.ts

```diff
@@ -27,6 +28,19 @@ export type UseCartFactoryParams<CART, CART_ITEM, PRODUCT, COUPON> = {
     currentCart: CART;
   }) => Promise<{ updatedCart: CART; updatedCoupon: COUPON }>;
   isOnCart: (params: { currentCart: CART; product: PRODUCT }) => boolean;
+  cartHelpers: {
+    getProducts: (cart: CART) => PRODUCT[];
+    getProductName: (product: PRODUCT) => string;
+    getProductImage: (product: PRODUCT) => string;
+    getProductPrice: (product: PRODUCT) => string;
+    getProductQty: (product: PRODUCT) => string;
+    getProductAttributes:
+      (product: PRODUCT, filterByAttributeName?: Array<string>) => Record<string, string | AgnosticProductAttribute>;
+    getProductSku: (product: PRODUCT) => string;
+    getTotals: (cart: CART) => AgnosticTotals;
+    getShippingPrice: (cart: CART) => number;
+    getTotalItems: (cart: CART) => number;
```

Comment:
we need to allow here to add more helpers

Reply:
Now, this type is inferred core/interfaces

Reply:
To discuss in another task